api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -754,6 +792,9 @@ private void updateProfileForUser(DbUser user, Profile updatedProfile, Profile p
       throw new BadRequestException("Cannot delete Verified Institutional Affiliation.");
     }
 
+    // if the Institution's Display Name differs from that of the VerifiedInstitutionalAffiliation
```

Comment:
VerifiedInstitutionalAffiliation has a non-authoritative displayName, to make `get` more descriptive